stdlib.sh

```diff
@@ -551,17 +551,26 @@ layout_pipenv() {
 # "$direnv_layout_dir/python-$python_version".
 #
 layout_pyenv() {
-  local python_version=$1
-  local pyenv_python
-  pyenv_python=$(pyenv root)/versions/${python_version}/bin/python
-  if [[ -x "$pyenv_python" ]]; then
-    if layout_python "$pyenv_python"; then
-      export PYENV_VERSION=$python_version
+  unset PYENV_VERSION
```

Comment:
can you add `local j` to avoid leaking the variable as well?

Reply:
done!